docs/docs/guides/assets/multi-stage-delivery/promote.yaml

```diff
@@ -0,0 +1,46 @@
+name: promote
+
+on:
+  workflow_dispatch:
+    inputs:
+      traceParent:
+        description: 'OTEL parent trace'
+        required: false
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          # configure git client
+          git config --global user.email "<email address>"
+          git config --global user.name "<name>"
+
+          # create a new branch
+          git switch -c production/${{ github.sha }}
+
+          # promote the change
+          cp dev/values.yaml production/values.yaml
```

Comment:
should we refer to this part of the code when we apply the values file?

Reply:
i have added a more detailed description of how the promotion using the values.yaml files works in the description of the files above, so now it should be clearer how it works